packages/desktopjs/src/registry.ts

```diff
@@ -68,16 +68,18 @@ export function resolveContainer(param1?: boolean | any, param2?: any): Containe
     if (!force && container) {
         return container;
     }
-
-    for (const entry in registeredContainers) {
-        try {
-            if (registeredContainers[entry].condition(options)) {
-                return container = registeredContainers[entry].create(options);
-            }
-        } catch (e) {
-            console.error("Error resolving container '" + entry + "': " + e.toString());
+    let registration: ContainerRegistration;
+    try {
+        for (let i: number =0; i< registeredContainers.length; i++) { // tslint:disable-line
```

Comment:
What doesn't tslint like here?

Reply:
because it has a rule saying prefer for..in